swarm/network/stream/delivery.go

```diff
@@ -178,22 +94,15 @@ func (d *Delivery) handleRetrieveRequestMsg(ctx context.Context, sp *Peer, req *
 			log.Debug("ChunkStore.Get can not retrieve chunk", "peer", sp.ID().String(), "addr", req.Addr, "hopcount", req.HopCount, "err", err)
 			return
 		}
-		if req.SkipCheck {
-			syncing := false
-			osp.LogFields(olog.Bool("skipCheck", true))
+		syncing := false
+		osp.LogFields(olog.Bool("skipCheck", true))
```

Comment:
Should this osp log field be removed also?

Reply:
done